wsdlgo/encoder.go

```diff
@@ -22,7 +22,7 @@ import (
 	"strings"
 	"text/template"
 
-	"github.com/fiorix/wsdl2go/wsdl"
+	"../wsdl"
```

Comment:
This does not work, please keep the original import path. Go is very opinionated on this.